indexed_gzip/tests/test_zran.py

```diff
@@ -20,25 +20,81 @@
 
     pytestmark = pytest.mark.zran_test
 
+    def test_file_funcs(testfile):
+        ctest_zran.test_file_funcs(testfile)
+
+    def test_init(testfile):
+        for no_fds in (True, False):
```

Comment:
Initially I considered adding a `--no_fds` flag to the entire test suite, but thought this approach was better because `no_fds` only affects certain tests in `test_zran`.